Lib/importlib/util.py

```diff
@@ -29,8 +29,8 @@ def resolve_name(name, package):
     if not name.startswith('.'):
         return name
     elif not package:
-        raise ValueError(f'no package specified for {repr(name)} '
-                         '(required for relative module names)')
+        raise ImportError(f'no package specified for {repr(name)} '
```

Comment:
Does this need a documentation update in https://github.com/python/cpython/blob/master/Doc/library/importlib.rst for `resolve_name` regarding change in exception type?

Reply:
Thank you, I've updated mentioned documentation.

Reply:
And what does the language reference say (if anything)?

Reply:
I can't find reference that mention about raising `ValueError` in here https://github.com/python/cpython/blob/master/Doc/reference/import.rst or perhaps I am looking in wrong place?